apps/condo/domains/user/schema/User.js

```diff
@@ -214,6 +215,7 @@ const User = new GQLListSchema('User', {
         dvAndSender(),
         historical(),
         webHooked(),
+        searchUpdateByTrigger([{ targetSchema: 'Ticket', triggerPathsToFields: ['assignee.name', 'executor.name'], schemaGql: Ticket }]),
```

Comment:
it looks like a bad code design. Probably this should work automatically inside the `searchBy` plugin

Reply:
moved to global preprocessors

Reply:
Moved to global plugins